src/ert/job_queue/job_queue_node.py

```diff
@@ -358,7 +358,7 @@ def _log_kill_thread_stopping_status(self) -> None:
     DONE_STATES = [
         JobStatus.SUCCESS,
         JobStatus.IS_KILLED,
-        JobStatus.DO_KILL_NODE_FAILURE,
+        JobStatus.DO_KILL,
```

Comment:
I am puzzled that `DO_KILL` was not here already. Do you understand why?

Reply:
Me too, will investigate ...